tests/rptest/services/cluster.py

```diff
@@ -220,14 +221,18 @@ def wrapped(self: HasRedpanda, *args: Any, **kwargs: Any):
                 else:
                     # stop here explicitly to fail if stop times out, otherwise ducktape won't catch it
                     self.redpanda.stop()
+                usage_stats = {}
+                for r in all_redpandas(self):
+                    usage_stats[r.who_am_i()] = r.usage_stats_dict
+                test_results["usage_stats"] = usage_stats
```

Comment:
I wouldn't be able to resist the temptation of recuding it into the following, but it's entirely up to you.
```
test_results["usage_stats"] = {r.who_am_i(): r.usage_stats_dict for r in all_redpandas(self)}
```